conf/application.conf

```diff
@@ -116,6 +116,17 @@ api {
     auth-validation.enabled = true
   }
 
+  supporting-agent-endpoints {
+    amend-annual-submission = true
```

Comment:
These are a bit shuffled, could they be grouped/ordered to make it easier to scan through them?
Maybe put the "annual submission" ones before all the "period summary" ones

(Basically what you see in the routes files)